README.md

```diff
@@ -23,10 +28,11 @@ Here is a simple sample configuration:
 
 ```yaml
 cloud:
-  k8s:
-    servicedns: ${SERVICE_DNS}
+  kubernetes:
```

Comment:
These options look like from later version? Can you update please to reflect the 1.3.x config?